katalog/prometheus-operated/deploy.yml

```diff
@@ -41,7 +42,10 @@ spec:
         resources:
           requests:
             storage: 50Gi
```

Comment:
I would increase this to at least `150Gi` because we always end up patching this value. What do you think?

cc @angelbarrera92 @phisco

Reply:
What will be the effect during an upgrade?

Reply:
Almost all of our deployment had this point patched, but I agree that is something to mention in the release notes and to check during the upgrade.

Anyway, we can setup a test case to assess what happens during the upgrade.